infra/terraform/global/iam.tf

```diff
@@ -25,3 +25,76 @@ EOF
 resource "aws_iam_access_key" "auth0_ses" {
     user = "${aws_iam_user.auth0_ses.name}"
 }
+
+resource "aws_iam_role" "softnas" {
```

Comment:
I wonder if we shouldn't have separate roles for the different environments. We usually do for other modules, it may be overkill in this case.